lib/Xud.ts

```diff
@@ -74,25 +75,30 @@ class Xud extends EventEmitter {
       this.db = new DB(loggers.db, this.config.dbpath);
       await this.db.init(this.config.initdb);
 
+      const swapClients: { [currency: string]: BaseClient | undefined } = {};
```

Comment:
I'm thinking it actually might have made sense to do this earlier, but I'm thinking it might be cleaner to make this a `Map<string, BaseClient>` type. We're mapping `currency` to other values elsewhere in the codebase. But this change could also happen in a separate PR.